conf/vary.routes

```diff
@@ -2,3 +2,5 @@
 
```

Comment:
would be good to refactor this and move to app.route

Reply:
Done with package changes. please verify and merge